### PR TITLE
Add project intern post

### PR DESCRIPTION
**Pull request by donjar**

### CODE REVIEW

**Comment by donjar** [comment]
@sushinoya changed

File: content/post/2017-09-15-friday-hacks-137-Sep-15.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Friday Hacks #137, September 15"
+date: 2017-09-12 09:40:35.299431
+author: Herbert
+---
+
+This Friday, our very own coreteam members will be haring their internship experiences both locally and overseas. We will also invite NUS students who have done internships in various companies.
```
Comment (jethrokuan) [inline content/post/2017-09-15-friday-hacks-137-Sep-15.md]: sharing*

File: content/post/2017-09-15-friday-hacks-137-Sep-15.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+{{% friday_hack_header venue="Seminar Room 1, NUS School of Computing (COM1-0206)" date="September 15" %}}
+
+### Internships, and How to Get Them
+
+#### Talk Description:
+
+Learn why you should do internships, how to choose, and of course, how to get them!
```
Comment (jethrokuan) [inline content/post/2017-09-15-friday-hacks-137-Sep-15.md]: choose __? companies?

File: content/post/2017-09-15-friday-hacks-137-Sep-15.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Ask questions of other students who have interned locally and overseas at various companies. This includes overseas like Google, Facebook, Twitter, Asana, and local companies like Google Singapore, Twitter Singapore, Carousell, Garena, and Tinkerbox!
```
Comment (jethrokuan) [inline content/post/2017-09-15-friday-hacks-137-Sep-15.md]: Ask questions of other students who have interned both locally and overseas. These companies include Google, Facebook, Twitter, Asana, as well as local companies like Google Singapore, Twitter Singapore, Carousell, Garena, and Tinkerbox!

File: content/post/2017-09-15-friday-hacks-137-Sep-15.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+You've probably read about tech internships and all their various perks. Heck, you've probaby even seen a movie about it. Now's the time to live it. This talk will address all the questions you've ever had about internships and help you on your way to getting one.
+
+Free pizza is served before the talks.
+
+#### Speaker Profile
+
+Varun is a Year 4 Computer Science student who did internships at Grab, Garena, Asana, and Jump Trading.
+
+Advay is a Year 3 Computer Science student who did internships at Xfers and Twitter.
```
Comment (jethrokuan) [inline content/post/2017-09-15-friday-hacks-137-Sep-15.md]: I forgot if the NUS enterprise stuff needs to be added in the post

Comment (donjar) [reply content/post/2017-09-15-friday-hacks-137-Sep-15.md]: We never did so far. :p we do on FB postings though.

Comment (advaypal) [reply content/post/2017-09-15-friday-hacks-137-Sep-15.md]: Can you change the order to Most recent first?

File: content/post/2017-09-15-friday-hacks-137-Sep-15.md (hunk 3 above)
Comment (advaypal) [reply content/post/2017-09-15-friday-hacks-137-Sep-15.md]: Add Apple as well

**Pull request merged**